src/ert/config/parsing/config_schema_deprecations.py

```diff
@@ -224,4 +224,16 @@
         "the future. Replace by NUM_CPU on a single compute node.",
         check=lambda line: "NUM_NODES" in line,
     ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="MEMORY_PER_JOB as QUEUE_OPTION to Torque is deprecated and will removed in "
```

Comment:
I see some of the other deprecation warnings output the queue system in all uppercase. I think that looks better as QUEUE_OPTION and the queue option itself are uppercase too.

Reply:
Fixed (but only for MEMORY*)